packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -240,6 +240,31 @@ export default declare((api, opts) => {
     }
   }
 
+  function hasMoreThanOneBinding(node) {
+    // ArrayPattern
+    if (node?.elements) {
+      if (node.elements.length > 1) return true;
```

Comment:
I just realize that `node.elements.length > 1` does not imply it has more than one binding, e.g.

```js
const [,,x] = arr
```
We should filter out the `null` nodes in `node.elements` and check that if the non-null elements have more than one element.

Reply:
Cool I'll put up a PR for that now

Reply:
Cool I'll put up a PR for that now

Reply:
Updated 😄